packages/vite/src/plugins/plugin.ts

```diff
@@ -583,3 +685,39 @@ function checkIfConfigFileShouldBeProject(
 
   return true;
 }
+
+async function getTestPaths(
```

Comment:
I don’t think this is accurate enough. It doesn’t take into account vitest’s projects syntax for test

Reply:
Thanks, done